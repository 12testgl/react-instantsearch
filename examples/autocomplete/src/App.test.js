import React from 'react';
import MultiIndex from './App-Multi-Index';
import Mentions from './App-Mentions';
import { render } from '@testing-library/react';

describe('autocomplete recipe', () => {
  it('MultiIndex renders without crashing', () => {
    const { container } = render(<MultiIndex />);

    expect(container).toMatchSnapshot();
  });
  it('Mentions renders without crashing', () => {
    const { container } = render(<Mentions />);

    expect(container).toMatchSnapshot();
  });
});
