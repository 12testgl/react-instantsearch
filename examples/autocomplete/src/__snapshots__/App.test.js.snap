// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`autocomplete recipe Mentions renders without crashing 1`] = `
<div>
  <div
    class="ant-mentions"
    style="width: 600px;"
  >
    <textarea
      class="rc-textarea"
      placeholder="give someone an @-mention here"
      rows="4"
    />
  </div>
</div>
`;

exports[`autocomplete recipe MultiIndex renders without crashing 1`] = `
<div>
  <div
    aria-expanded="false"
    aria-haspopup="listbox"
    aria-owns="react-autowhatever-1"
    class="react-autosuggest__container"
    role="combobox"
  >
    <input
      aria-autocomplete="list"
      aria-controls="react-autowhatever-1"
      autocomplete="off"
      class="react-autosuggest__input"
      placeholder="Search for a product..."
      type="text"
      value=""
    />
    <div
      class="react-autosuggest__suggestions-container"
      id="react-autowhatever-1"
      role="listbox"
    />
  </div>
</div>
`;
